e2e/cli_test.go

```diff
@@ -506,22 +487,22 @@ func TestCreateNewCustomRuleFromTemplate(t *testing.T) {
 func TestCreateNewBuiltinRuleFromTemplate(t *testing.T) {
 	t.Parallel()
 
+	if runtime.GOOS == "windows" {
+		t.Skip("temporarily skipping this test on Windows")
+	}
+
 	stdout := bytes.Buffer{}
 	stderr := bytes.Buffer{}
-
 	tmpDir := t.TempDir()
 
-	err := regal(&stdout, &stderr)("new", "rule", "--category", "naming", "--name", "foo-bar-baz", "--output", tmpDir)
+	expectExitCode(t, regal(&stdout, &stderr)(
+		"new", "rule", "--category", "naming", "--name", "foo-bar-baz", "--output", tmpDir,
+	), 0, &stdout, &stderr)
```

Comment:
Hmm I think this could go one step further? To let us write
```go
expectExitCode(t, 0, "new", "rule", "--category", "naming", "--name", "foo-bar-baz", "--output", tmpDir)
```
or something like that? 🤔 Deal with the buffers in the helper, etc.

Reply:
I did something like that in the first iteration, but the problem is that many of the tests then go on to check for the contents of the buffers, so we'll need them outside of the helper. There's definitely some improvements to make here though, and I hope to see more in later PRs :)